test/everything.sh

```diff
@@ -5,6 +5,8 @@ set -x
 
 srcdir=`dirname $0`/..
 
+$srcdir/test/test_tools.sh
```

Comment:
You don't need another shell file. Either put it as unit test (in index.js) or as functional test in cmdline.sh